src/main/kotlin/no/nav/dagpenger/regel/api/models/Behov.kt

```diff
@@ -45,10 +45,8 @@ data class InternBehov(
         fun toPacket(internBehov: InternBehov): Packet = Packet("{}").apply {
             this.putValue(PacketKeys.BEHOV_ID, internBehov.behovId.id)
             this.putValue(PacketKeys.AKTØR_ID, internBehov.aktørId)
-            when (internBehov.behandlingsId.regelKontekst.type) {
-                Kontekst.VEDTAK -> this.putValue(PacketKeys.VEDTAK_ID, internBehov.behandlingsId.regelKontekst.id)
-                Kontekst.CORONA -> this.putValue(PacketKeys.CORONA_ID, internBehov.behandlingsId.regelKontekst.id)
-            }
+            this.putValue(PacketKeys.VEDTAK_ID, internBehov.behandlingsId.regelKontekst.id)
```

Comment:
skal vi la denne hete `kontekstId` og la 

``` 
  when (internBehov.behandlingsId.regelKontekst.type) {
                Kontekst.VEDTAK -> this.putValue(PacketKeys.VEDTAK_ID, internBehov.behandlingsId.regelKontekst.id)
```

for å være bakoverkombatibel.

Reply:
ok